app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserMailer < ApplicationMailer
+  default from: "MrClean@mrclean.ca"
```

Comment:
This should go in the ApplicationMailer.

We should talk to Dan about getting an official `getyardstick.com` email address for Mr. Clean.